gatsby-config.js

```diff
@@ -207,13 +223,21 @@ module.exports = {
       options: {
         name: 'translated-content',
         path: `${__dirname}/src/i18n/content`,
+        ignore:
+          process.env.BUILD_I18N === 'false'
```

Comment:
nit-pick, but the not-logic always gets me, would the suggestion work?

```suggestion
          process.env.BUILD_I18N ? [] : [`${__dirname}/src/i18n/content/*`]
```

Reply:
i think env vars are always strings unfortunately so we would have to change it to a boolean first to do this 🤷‍♀️

Reply:
omg yeah my b